scripts/maint/checkSpace.pl

```diff
@@ -177,7 +177,7 @@ sub msg {
                     $1 ne "elsif" and $1 ne "WINAPI" and $2 ne "WINAPI" and
                     $1 ne "void" and $1 ne "__attribute__" and $1 ne "op" and
                     $1 ne "size_t" and $1 ne "double" and $1 ne "uint64_t" and
-                    $1 ne "workqueue_reply_t") {
+                    $1 ne "workqueue_reply_t" and $1 ne "bool") {
```

Comment:
This list of keywords, types, etc. is getting rather unwieldy to have in the expression of an if-statement. Maybe make a list or something?

Reply:
Opened trac#30990 for this.